### PR TITLE
[pt-br] add content/pt-br/docs/tasks/configure-pod-container/translate-compose-kubernetes.md

### PR DESCRIPTION
**Pull request by DonatoHorn**

### CODE REVIEW

**Comment by DonatoHorn** [comment]
Hi, there.
After about 12 months of waiting for revision, this PR wasn't up to date with the latest version of de docs, so I updated the file to follow the latest version.

File: content/pt-br/docs/tasks/configure-pod-container/translate-compose-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+INFO OpenShift file "foo-buildconfig.yaml" created
+```
+
+{{< note >}}
+Se você está empurrando manualmente os artefatos do OpenShift usando ``oc create -f``, é preciso garantir que você empurre o artefato `imagestream` antes do artefato do `buildconfig`, para contornar esta questão do OpenShift: https://github.com/openshift/origin/issues/4518 .
+{{< /note >}}
+
+## Conversões alternativas
+
+A transformação padrão do `kompose` vai gerar [Implantações](/docs/concepts/workloads/controllers/deployment/) Kubernetes e [Serviços](/docs/concepts/services-networking/service/), em formato yaml. Você tem uma opção alternativa para gerar json com `-j`.
```
Comment (edsoncelio) [inline content/pt-br/docs/tasks/configure-pod-container/translate-compose-kubernetes.md:340]: Implantações] -> Deployments 
[Serviços] -> Services

File: content/pt-br/docs/tasks/configure-pod-container/translate-compose-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+```
+
+{{< note >}}
+Se você está empurrando manualmente os artefatos do OpenShift usando ``oc create -f``, é preciso garantir que você empurre o artefato `imagestream` antes do artefato do `buildconfig`, para contornar esta questão do OpenShift: https://github.com/openshift/origin/issues/4518 .
+{{< /note >}}
+
+## Conversões alternativas
+
+A transformação padrão do `kompose` vai gerar [Implantações](/docs/concepts/workloads/controllers/deployment/) Kubernetes e [Serviços](/docs/concepts/services-networking/service/), em formato yaml. Você tem uma opção alternativa para gerar json com `-j`. 
+Além disso, você pode gerar alternativamente objetos [Controladores de Replicação](/docs/concepts/workloads/controllers/replicationcontroller/), [Daemon Sets](/docs/concepts/workloads/controllers/daemonset/), ou `charts` [Helm](https://github.com/helm/helm).
```
Comment (edsoncelio) [inline content/pt-br/docs/tasks/configure-pod-container/translate-compose-kubernetes.md:341]: [Daemon Sets] -> DaemonSets

**Review by edsoncelio — CHANGES_REQUESTED** [review]

**Comment by edsoncelio** [comment]
Hey @DonatoHorn! 
Due to the lack of response, we will close this PR but feel free to reopen it If you want to make the requested changes.
/close

**Pull request closed without merging**